client/post-editor/media-modal/index.jsx

```diff
@@ -241,9 +243,28 @@ export class EditorMediaModal extends Component {
 		this.props.setView( ModalViews.DETAIL );
 	};
 
-	onImageEditorCancel = imageEditorProps => {
-		const { mediaLibrarySelectedItems } = this.props;
+	handleUpdatePoster = ( { ID, posterUrl } ) => {
+		const { site } = this.props;
+
+		// Photon does not support URLs with a querystring component.
+		posterUrl = posterUrl && posterUrl.split( '?' )[ 0 ];
```

Comment:
This is fixed in https://github.com/Automattic/wp-calypso/pull/11653/commits/624c224d1d49b83a4c22f254b4e688b3ff26d418.